src/components/Table/README.md

```diff
@@ -263,7 +263,7 @@ const MyTable1 = withTableSettings({sortable: false})(Table);
 ```ts
 type TableSettingsData = Array<{
   id: string;
-  isSelected?: boolean;
+  selected?: boolean;
```

Comment:
Why this change? It's a breaking change

Reply:
agree, need to separate `TableColumnSetupItem` from `TableSettingsData` and return backward compatibility for `TableColumnSetupItem` only